crates/coresimd/src/lib.rs

```diff
@@ -46,6 +46,50 @@ extern crate stdsimd_test;
 #[cfg(test)]
 extern crate test;
 
+#[doc(hidden)]
+macro_rules! cfg_if {
```

Comment:
Since this won't be available in libcore can this be moved diretly to the coresimd/mod.rs file to be used there?

Reply:
Sure :)

Reply:
Done :)